### PR TITLE
splash screen adapt for alipay&cocos play

### PR DESCRIPTION
**Pull request by JayceLai**
Re: cocos-creator/3d-tasks#3567

Changes:
 * splash screen adapt for alipay&cocos play

### CODE REVIEW

File: cocos/core/splash-screen-webgl.ts [hunk 1]
```diff
@@ -125,8 +126,9 @@ export class SplashScreenWebgl {
     }
 
     public main (canvas: HTMLCanvasElement) {
-        if (window._CCSettings && window._CCSettings.splashScreen) {
-            this.setting = window._CCSettings.splashScreen;
+        this.ccSetting = window._CCSettings;
```
Comment (pandamicro) [inline cocos/core/splash-screen-webgl.ts]: Please do not save CCSettings, it's too big, only save what you need here. By the way, why you need to save it ?

File: cocos/core/splash-screen-webgl.ts [hunk 2]
```diff
@@ -221,30 +223,45 @@ export class SplashScreenWebgl {
...
-            if (sys.os == cc.sys.OS_OSX || sys.os == cc.sys.OS_IOS) {
+            if (sys.os == sys.OS_OSX || sys.os == sys.OS_IOS) {
                 this.gl.canvas.width = screen.width * devicePixelRatio;
                 this.gl.canvas.height = screen.height * devicePixelRatio;
             }
         }
 
-        // TODO: hack for cocosPlay & XIAOMI cause on landscape canvas value is wrong
+        // adapt for cocos play 1.0.11
+        if (COCOSPLAY) {
```
Comment (pandamicro) [inline cocos/core/splash-screen-webgl.ts:234]: There is no version distinguish between this if and the next one, it's not acceptable to run the lame adaptation code  twice for the same platform.

File: cocos/core/splash-screen-webgl.ts [hunk 3]
```diff
@@ -385,13 +402,23 @@ export class SplashScreenWebgl {
 
         gl.bindFramebuffer(gl.FRAMEBUFFER, null);
 
-        // TODO: hack for cocosPlay & XIAOMI cause on landscape canvas value is wrong
+        // adapt for cocos play 1.0.11
+        if (COCOSPLAY) {
```
Comment (pandamicro) [inline cocos/core/splash-screen-webgl.ts]: Again ? Why keep set innerWidth and innerHeight ?

Comment (JayceLai) [reply cocos/core/splash-screen-webgl.ts]: To prevent someone silently modifying

Comment (pandamicro) [reply cocos/core/splash-screen-webgl.ts]: In which case exactly ? I don't understand the scenario where we need to check and adapt window size every frame.

Comment (JayceLai) [reply cocos/core/splash-screen-webgl.ts]: @LinYunMo and I fixed the problem on 1.0.3, https://github.com/cocos-creator/engine/pull/6107

Not sure if it's needed now
I will test the check is needed or not

Comment (JayceLai) [reply cocos/core/splash-screen-webgl.ts]: It is unnecessary on 1.1.1

**Review by pandamicro — APPROVED** [review]

**Pull request merged**